.travis.yml

```
@@ -17,6 +17,7 @@ install:
   - npm install
   - npm run webpack
   - pip install --upgrade . -r helm-chart/images/binderhub/requirements.txt
+  - pip install -U "docker~=4.2.0"
```

I'm adding a comment about what this relates to so we can understand that this is a temporary workaround later.

```suggestion
  # NOTE: The latest docker python package (4.3.0) requires a more modern docker
  #       version (newer than 18.06.0-ce / API version: 1.38) which is not yet
  #       available on travis.
  #       ref: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1743
  - pip install -U "docker~=4.2.0"
```